index.php

```diff
@@ -78,34 +120,20 @@
                 </div>
 
                 <table class="tasks">
-                    <tr class="tasks__item task">
+                    <?php foreach($tasks as $key => $val) :
```

Comment:
Лучше именовать понятно. 
```suggestion
                    <?php foreach($tasks as $key => $task) :
```
А ещё `$key` не используется нигде.